lighthouse-core/audits/bootup-time.js

```diff
@@ -159,6 +159,7 @@ class BootupTime extends Audit {
 
     const summary = {wastedMs: totalBootupTime};
 
+    /** @type {LH.Audit.Details.Table['headings']} */
```

Comment:
had to add a ton of these explicit `headings` type lines because the heading `itemType` now has a more stringent type (union of literals instead of just `string`) which won't work if it's been widened. There's an upcoming tsc 3.4 feature (`as const`) that will make this a bit easier all over the codebase, but this doesn't seem too bad for now